### ISSUE TITLE
Using the script without a token

### ISSUE THREAD

**Issue by shoopdawoop** [alex-phillips/plex-autocollections#41]
is it possible to use this script with a server on the local network (or on localhost) without ever involving a plex.tv account and an API token?

**Comment by alex-phillips** [comment]
@shoopdawoop we use PlexAPI for authentication. So whatever they support this script should support as well. If you find that a method doesn't work, let me know! https://pypi.org/project/PlexAPI/

**Comment by shoopdawoop** [comment]
I realize I did not explain my question well enough, but I figured it out and made it work now, so here is my conclusion in case it might be helpful for others:

I am using a Plex Media Server on a LAN only setup with no plex.tv account, no user login, and `allowedNetworks` set to 192.168.x.x/24 so authorization is turned off completely for that entire network.

While the PlexAPI docs state to either provide login+pass or get an access token from MyPlex, in my case it is enough to provide a fake token, and I had to add the http:// schema as part of the URL in main.py which I was missing before, like so

```
PLEX_URL = 'http://localhost:32400'
PLEX_TOKEN = 'lulz'
```

(https:// would probably work fine but I am using a self-signed certificate).

Now I have got a pile of YouTube videos nicely sorted in collections.


Thank you!
